src/lib/firestore/post.ts

```diff
@@ -0,0 +1,62 @@
+import { DocumentData, QueryDocumentSnapshot } from 'firebase-admin/firestore'
+import moment from 'moment'
+
+import { db } from './firestore'
+
+import { postStatus } from '@/consts/constants'
+import { Post } from '@/types/Post'
+
+export const GetPosts = async () => {
```

Comment:
どんなPostを取ってるか明示しておきましょう。
```suggestion
export const GetApprovedPosts = async () => {
```